### ISSUE TITLE
Version 2.1.1 breaks webpack-offline-plugin

### ISSUE THREAD

**Issue by jefbarn** [survivejs/webpack-merge#55]
Updated to 2.1.1 and getting error in webpack-offline-plugin now..
```
TypeError: #<Object> is not a function
    at Array.map (native)
    at OfflinePlugin.validatePaths (C:\projects\dayapp\node_modules\offline-plugin\lib\index.js:671:21)
    at OfflinePlugin.setAssets (C:\projects\dayapp\node_modules\offline-plugin\lib\index.js:339:29)
    at Compiler.<anonymous> (C:\projects\dayapp\node_modules\offline-plugin\lib\index.js:288:18)
...
```

Reverting to 2.1.0 and error goes away.

**Comment by bebraw** [comment]
Yeah, it's the prototype thing. I think I'll revert that feature and rethink the approach. Thanks.

**Comment by bebraw** [comment]
Should be fixed in `2.2.0`. My apologies for the inconvenience.
